src/containers/sign-on/store/sagas.js

```diff
@@ -468,7 +467,11 @@ function* followArtists() {
     // The update user location depends on the user being discoverable in discprov
     // So we wait until both the user is indexed and the follow user actions are finished
     yield call(AudiusBackend.updateUserLocationTimezone)
-    yield call(fetchAccountAsync, { fromSource: true })
+
```

Comment:
Second important change. We don't want to call `fetchAccountAsync` again here after following users because the aggregate_user view in discovery node will probably not have updated (yet).

Instead, we want to make sure that after we've followed everyone, our local storage persisted user is refreshed with the changes from `adjustUserField` that implicitly got called by all these follows

Reply:
yeah. before the aggregate views (/ or got slow), this was not an issue because refetching would have the counts updated